src/client/src/components/DraggableSidebarItem/styles.module.css

```diff
@@ -47,6 +47,10 @@
   margin-left: 10px;
 }
 
+.pagesTextContainerServices {
```

Comment:
Is this class used somewhere?

Reply:
Oh yeah this was from an old class that I deleted but forgot to remove the styling for. Good catch!

Reply:
same with the one on the bottom - I will remove both